manifest.json

```diff
@@ -41,56 +45,12 @@
     },
     {
       "name": "is_public",
+      "type": "boolean",
       "ask": {
         "en": "Is it a public Shaarli site ?",
         "fr": "Est-ce un site Shaarli public ?"
       },
-      "choices": ["Yes", "No"],
-      "default": "No"
-    },
-    {
-      "name": "privatelinkbydefault",
-      "ask": {
-        "en": "Are new links private by default ?",
-        "fr": "Les nouveaux liens sont-ils privés par défaut ?"
-      },
-      "choices": ["Yes", "No"],
-      "default": "Yes"
-    },
-    {
-      "name": "privateinstance",
-      "ask": {
-        "en": "Is this instance private?",
-        "fr": "Cette instance est-elle privée ?"
-      },
-      "choices": ["Yes", "No"],
-      "default": "Yes"
-    },
-    {
-      "name": "admin",
-      "type": "user",
-      "ask": {
-        "en": "Owner of the Shaarli instance",
-        "fr": "Propriétaire de l'instance Shaarli"
-      },
-      "example": "test"
-    },
-    {
-      "name": "password",
-      "type": "password",
-      "ask": {
-        "en": "Define password for Shaarli user",
-        "fr": "Définissez le mot de passe de l'utilisateur Shaarli"
-      }
-    },
-    {
-      "name": "title",
-      "ask": {
-        "en": "Choose a title for Shaarli's page",
-        "fr": "Choissez un titre pour la page Shaarli"
-      },
-      "example": "Shaarli",
-      "default": "Shaarli"
+      "default": true
```

Comment:
I think both the option are default by YunoHost. Better option is to implement Fail2Ban on Shaarli.
Go ahead if you want to change to non-default.

Reply:
> Better option is to implement Fail2Ban on Shaarli.

I wasn't saying that in terms of security.
I mean, most users (at least a good part of them ?) use this as a personal and private instance.
I would prefer to keep Shaarli private by default, and make it public only if some conscious users (who want to publicly share their content) choose that during the installation.

Reply:
Really go ahead. I have no opinion about it.Till the Cl is happy, its good for me.

Reply:
Ok.
If I just put Yes as default, would it work correctly or should I stick to "false" ?

Reply:
It will take either "true" or "false". This has been changed by the app team and I don't have solution for "yes" or "no". If you want to do it, you will have to remove the type Boolean field and change the code accordingly in all the scripts.